core/templates/dev/head/pages/exploration_editor/ExplorationWarningsService.js

```diff
@@ -173,11 +179,31 @@ oppia.factory('ExplorationWarningsService', [
       return results;
     };
 
+    var _getStatesWithUnresolvedAnswers = function() {
```

Comment:
Hm -- this name is misleading. Maybe _getStatesWithAnswersThatMustBeResolved()?

Reply:
Done.